src/main/scala/nl.knaw.dans.easy.solr4files/components/AuthorisationComponent.scala

```diff
@@ -32,13 +32,15 @@ trait AuthorisationComponent extends DebugEnhancedLogging {
 
   trait Authorisation {
     val baseUri: URI
+    val connectionTimeOut: Int // ms
```

Comment:
@rvanheest : why not just that bit of documentation ("ms") in the name? ("Self-documenting code" :0)

Reply:
great suggestion! I noticed that the rest of the names I added already has that `ms` in it. Not sure why I didn't for this instance...
See 15ef3ac